docs/inputdata.md

```diff
@@ -15,6 +17,34 @@ and we continue to ingest them as before.
 
 ## Source Files
 
+*Starting with Unicode 15.1, the “source of truth” for most data files is in the repo,
+and most of this section is obsolete. See [data workflow](data-workflow.md).
+The biggest exception is Unihan.zip, which we don't track in the repo; see the Unihan section below.
+Also, it's still useful to delete the BIN files/folders after changing data files.*
+
+### Unihan
```

Comment:
FYI: I moved this section, unchanged, up from below. This Unihan section is still relevant. The other ones only remain for historical information (nostalgia?).